### PR TITLE
Add ORFanage

### PR DESCRIPTION
**Pull request by alevar**
Add ORFanage method. ORFanage is an efficient and sensitive method to search for ORFs in spliced genomes guided by reference annotation to maximize protein similarity within genes.

### CODE REVIEW

**Comment by alevar** [comment]
@BiocondaBot please add label

File: recipes/orfanage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.4_9f713d2" %}
+{% set commit_id = "9f713d2a55065bb952c29eee1272ad9a9ca09ee2" %}
+
+package:
+  name: "orfanage"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/alevar/ORFanage.git
```
Comment (bgruening) [inline recipes/orfanage/meta.yaml:9]: can you create a tarball release and then use that one?

File: recipes/orfanage/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - zlib
+    - libbigwig
+
+test:
+  commands:
+      - orfanage --help > /dev/null
+
+about:
+  home: https://github.com/alevar/ORFanage
+  license: GPLv3
```
Comment (bgruening) [inline recipes/orfanage/meta.yaml:35]: can you please use a SPDX identifier for a license?

File: recipes/orfanage/meta.yaml (hunk 1 above)
Comment (alevar) [reply recipes/orfanage/meta.yaml]: Thank you for reviewing my PR. After adding a tarball release  inplace of git_url I get the following error during Azure tests. Am I doing something wrong?

Error was:
Command '['/opt/conda/bin/git', 'clone', '--mirror', 'https://github.com/alevar/ORFanage/archive/refs/tags/1.0.4.tar.gz', '/opt/conda/conda-bld/git_cache/github.com/alevar/ORFanage/archive/refs/tags/1.0.4.tar.gz']' returned non-zero exit status 128.

Thank you,

Comment (bgruening) [reply recipes/orfanage/meta.yaml]: You can not clone a tarball, you need to use `url:`
Please read more in our documentation at: https://bioconda.github.io/contributor/index.html

Comment (alevar) [reply recipes/orfanage/meta.yaml]: Thank you. I have updated the source and meta to comply with request.

File: recipes/orfanage/meta.yaml (hunk 2 above)
Comment (alevar) [reply recipes/orfanage/meta.yaml]: Thank you. License tag is updated

**Comment by alevar** [comment]
@BiocondaBot please add label

File: recipes/orfanage/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+      - orfanage --help > /dev/null
+
+about:
+  home: https://github.com/alevar/ORFanage
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  summary: 'Ultra-efficient and sensitive method to search for ORFs in spliced genomes guided by reference annotation to maximize protein similarity within genes.'
+
+extra:
```
Comment (bgruening) [inline recipes/orfanage/meta.yaml:39]: This can now be removed.

File: recipes/orfanage/build.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# flags for building libBigWig dependency
+#export C_INCLUDE_PATH=${PREFIX}/include
+#export LIBRARY_PATH=${PREFIX}/lib
+#export LD_LIBRARY_PATH=${PREFIX}/lib
+
+#CFLAGS="$CFLAGS -g -Wall -O3 -Wsign-compare -L$PREFIX/lib -I$PREFIX/include"
+#LIBS="$LDFLAGS -L$PREFIX/lib -lm -lz"
+
```
Comment (bernt-matthias) [inline recipes/orfanage/build.sh:10]: Commented code may be removed.

File: recipes/orfanage/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    - llvm-openmp  # [osx]
+  host:
+    - gsl
+    - libcblas
+    - zlib
+    - libbigwig
+
+test:
+  commands:
+      - orfanage --help > /dev/null
```
Comment (bernt-matthias) [inline recipes/orfanage/meta.yaml:31]: Consistent 2 spaces indent.

**Review by bernt-matthias — COMMENTED** [review]
Just minor ones.

**Review by bernt-matthias — APPROVED** [review]

**Pull request merged**